### ISSUE TITLE
A11y_SQLAzureDataStudio_DeployDacpac_ReviewDeployPlan_Keyboard:The scroll bar is not accessible.

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#9268]
[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11y_SQLAzureDataStudio;#A11yMAS;#A11ySev1;#A11yTCS;#SQL Azure Data Studio;#Benchmark;#DesktopApp;#Win32;#MAS2.1.1;#Keyboard;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.16.0-insider
Commit: 506c6a5e5f8423d3249670f55b1466b1980a5f03
Date: 2020-02-17T05:43:18.399Z
VS Code: 1.42.0
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18363

**Additional Details:**
MAS References:MAS2.1.1

**Repro Steps:**
1. Launch Azure Data Studio application.
2. Connect to server with windows credentials.
3. Navigate to Extensions from left menu.
4. Search & install Dacpac extension.
5. Navigate to Connections from left menu.
6. Expand the server connected in step 2.
7. Right click an on prem database and select Data-tier application wizard from Context menu.
8. Select first radio button.
9. Navigate till next button & hit enter.
10. Provide all the details for Deploy Dacpac & hit next.
11. Navigate to the table using tab.

**Actual:**
On navigating on the page the tab focus after 3 is on the checkbox, the scroll bar is not accessible hence the data in the table will not be accessible to keyboard users.

**Expected:**
The scroll bar should be accessible after line 3 on the tab.

**User Impact:**
Keyboard users will be able to view the data in the table.

**Attachment link for Reference :**
https://microsoft-my.sharepoint.com/:p:/g/personal/v-stfe_microsoft_com1/EbE-foRNRtlHjEcW5sSEMIQB10kFj9NCaQFolAJm9HlE3A?e=VxDN0Q

**Comment by alanrenmsft** [comment]
I don't think the scroll bar needs to be accessible by keyboard, you can use the table navigation keys to navigate inside the table. this is by design.

**Comment by sspatil123** [comment]
GithubTags:#Closed;Version 1.42.0
[9268_Fixed.webm](https://user-images.githubusercontent.com/85863999/218075219-2933f4b7-cb90-4854-a465-6ed3f5aecb3a.webm)

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-Win32-Feb2020;
